libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.html

```diff
@@ -25,6 +28,7 @@
     [showAddButton]="skyComponentProperties?.showAddButton"
     [showMoreConfig]="skyComponentProperties?.showMoreConfig"
     [wrapperClass]="skyComponentProperties?.wrapperClass"
+    (addClick)="skyComponentProperties?.addClick($event)"
```

Comment:
Do we have unit tests that validate that these properties are passed through as expected? If not, we should have one; if we already have one, we should add these to that validation.

Reply:
Good call. Added tests.